types/braintree/index.d.ts

```diff
@@ -0,0 +1,1749 @@
+        createdAt: Date;
+        expired: boolean;
+        subscriptions?: Subscription[];
+        uniqueNumberIdentifier: string;
+        updatedAt: Date;
+    }
+
+    /**
+     * PayPal Account
+     */
+
+    export interface PayPalAccount {
+        imageUrl: string;
+        payerId: string;
+        token: string;
+        billingAgreementId: string;
+        createdAt: Date;
+        customerId: string;
+        default: boolean;
+        email: string;
+        revokedAt: string;
+        subscriptions?: Subscription[];
+        updatedAt: Date;
+    }
+
+    /**
+     * Samsung Pay Card
+     */
+
+    export interface SamsungPayCard {
+        bin: string;
+        cardType: string;
+        cardholderName: string;
+        commercial: Commercial;
+        countryOfIssuance: string;
+        customerLocation: CustomerLocation;
+        debit: Debit;
+        durbinRegulated: DurbinRegulated;
+        expirationDate: string;
+        expirationMonth: string;
+        expirationYear: string;
+        healthcare: HealthCare;
+        imageUrl: string;
+        issuingBank: string;
+        last4: string;
+        maskedNumber: string;
+        payroll: Payroll;
+        prepaid: Prepaid;
+        productId: string;
+        sourceCardLast4: string;
+        token: string;
+        billingAddress: Address;
+        createdAt: Date;
+        customerId: string;
+        expired: boolean;
+        subscriptions?: Subscription[];
+        uniqueNumberIdentifier: string;
+        updatedAt: Date;
+    }
+
+    /**
+     * Venmo Account
+     */
+
+    export interface VenmoAccount {
+        imageUrl: string;
+        sourceDescription: string;
+        token: string;
+        username: string;
+        venmoUserId: string;
+        createdAt: Date;
+        customerId: string;
+        default: boolean;
+        subscriptions?: Subscription[];
+        updatedAt: Date;
+    }
+
+    /**
+     * Visa Checkout Card
+     */
+
+    export interface VisaCheckoutCard {
+        billingAddress: Address;
+        bin: string;
+        callId: string;
+        cardType: string;
+        cardholderName: string;
+        commercial: Commercial;
+        countryOfIssuance: string;
+        createdAt: Date;
+        customerId: string;
+        customerLocation: CustomerLocation;
+        debit: Debit;
+        default: boolean;
+        durbinRegulated: DurbinRegulated;
+        expirationDate: string;
+        expirationMonth: string;
+        expirationYear: string;
+        expired: boolean;
+        healthcare: HealthCare;
+        imageUrl: string;
+        issuingBank: string;
+        last4: string;
+        maskedNumber: string;
+        payroll: Payroll;
+        prepaid: Prepaid;
+        productId: string;
+        subscriptions?: Subscription[];
+        token: string;
+        uniqueNumberIdentifier: string;
+        updatedAt: Date;
+    }
+
+    /**
+     * Payment Method Enums
+     */
+
+    export enum Commercial {
+        Yes,
```

Comment:
I'd check what the actual value of the enums throughout this are (whether they are string enums or actually numbers).

Reply:
Yeah, missed this and will fix!